source/includes/v2/_deprecated.md

```diff
@@ -0,0 +1,50 @@
+# Deprecated endpoints
+
+The following endpoints are deprecated and will be removed in a future release.
+
+
+## Catalog aggregates
+`GET /api/v2/catalog/aggregates`
+
+
+<aside>
+Will be removed in september 2021.
```

Comment:
The Months of the Year are always capitalized. Sorry, I missed this issue in my first review.
```suggestion
Will be removed in September 2021.
```